edit_course/operations/configure.py

```diff
@@ -496,11 +496,14 @@ def get_target_category(category, course=None, course_url=None):
             return None, _("Course: {}, Instance: {}, not found.".format(course_name, instance_name))
 
     else:
-        course_slug_begin = 0
-        if not settings.BASE_URL in course_url:
-            return None, _('ExerciseColletion URL "{}" not in correct domain {}.'.format(course_url,settings.BASE_URL))
 
-        course_slug_begin = len(settings.BASE_URL)
+        course_slug_begin = course_url.find('/', course_url.find('://') + 1) + 1
```

Comment:
Would it be more clear using urllib.parse.urlsplit here?

Somethin like:

```
prefix = urlsplit(settings.BASE_URL).path.rstrip('/') + '/'
path = urlsplit(course_url).path.rstrip('/')
if not path.startswith(prefix):
  raise Error("Invalid prefix")
course, instance = path[len(prefix):].split('/', 1)
```
In addition, we should probably validate domain too?